cve_bin_tool/util.py

```diff
@@ -147,7 +151,7 @@ def __init__(
         self.yield_files = yield_files
         self.yield_folders = yield_folders
 
-    def walk(self, roots=None):
+    def walk(self, roots: Optional[List[str]] = None):
```

Comment:
Simplified due to `annotations` imported above (and also added return type):
```suggestion
    def walk(self, roots: list[str] | None = None) -> Iterator[str]:
```